wasm2c/examples/fac/fac.c

```diff
@@ -586,13 +591,13 @@ DEFINE_TABLE_SET(externref)
 DEFINE_TABLE_FILL(funcref)
 DEFINE_TABLE_FILL(externref)
 
-static bool s_module_initialized = false;
-
-static u32 func_types[1];
+#if defined(__GNUC__) || defined(__clang__)
+#define FUNC_TYPE_T(x) static const char* const x
+#else
+#define FUNC_TYPE_T(x) static const char x[]
+#endif
```

Comment:
Can we not just use `static const char*` in both cases, avoiding the macro completely?

Reply:
Unfortunately I couldn't figure out how -- on MSVC, a `static const char* const` doesn't seem to be a constant expression and can't be used in an initializer, which is how we represent elem segments. (On the other hand, I think C requires a `static const char[]` to have a distinct address so it can't be merged with another identical string constant.)